tools/mcl/mcl.xml

```diff
@@ -1,5 +1,5 @@
 <tool id="mcl_clustering" name="MCL"  version="14.137">
-    <description>A Markov Cluster Algorithm.</description>
+    <description>A Markov Cluster Algorithm</description>
```

Comment:
Why do you touch mcl in this PR. I see travis is failing because of these tests. MCL does not have yet a conda recipe, at least not in this version?